scripts/genbuild.py

```
@@ -1,9 +1,11 @@
 #!/usr/bin/python3
 
 import semver
-current = open('MiAZ/data/docs/VERSION', 'r').read().strip()
+
+FILEPATH_VERSION = 'data/docs/VERSION'
+current = open(FILEPATH_VERSION, 'r').read().strip()
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)